layouts/shortcodes/youtubetest.html

```diff
@@ -0,0 +1,5 @@
+{{ $id    := .Get 0 }}
```

Comment:
Should the file be called youtubetest.html? 😬

Reply:
lulz. Fixed.